toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -117,7 +176,12 @@ func CreateInstallRoot(installRoot string, mountPointMap, mountPointToMountArgsM
 // DestroyInstallRoot unmounts each of the installroot mountpoints in order, ensuring that the root mountpoint is last
 // - installRoot is the path to the root where the mountpoints exist
 // - mountPointMap is the map of mountpoints to partition device paths
-func DestroyInstallRoot(installRoot string, mountPointMap map[string]string) (err error) {
+// - mountPointToOverlayMap is the map of mountpoints to overlay devices
+func DestroyInstallRoot(installRoot string, mountPointMap map[string]string, mountPointToOverlayMap map[string]*Overlay) (err error) {
+	logger.Log.Trace("Destroying InstallRoot")
+
+	defer OverlayUnmount(mountPointToOverlayMap)
```

Comment:
Do we want to capture and return an error with this `OverlayUnmount` if it fails?

Reply:
Discussed offline, this is non-fatal.